developers_chamber/ecs_utils.py

```diff
@@ -397,13 +399,14 @@ def run_service_task(cluster, service, command, success_string, timeout, region,
     resp = ecs_client.describe_task_definition(taskDefinition=latest_task_definition)
     container_definitions = resp['taskDefinition']['containerDefinitions']
 
-    if len(container_definitions) != 1:
+    container_definition = [d for d in container_definitions if d["name"] == container]
+
+    if len(container_definition) != 1:
         raise ClickException(
-            ('Exactly one container is allowed to be specified in service.\n'
-             'Number of containers specified: {}'.format(len(container_definitions)))
-        )
+            ('Container "{}" not found in task.\n'
+             'Containers available in task: {}'.format(container, [d["name"] for d in container_definitions])))
```

Comment:
The error might be confusing, because there might be more than one definition as well, but I guess it doesn't matter?

Reply:
This function executes a command by running (ecs) task and overriding the default behaviour (e.g. instead starting uwsgi, you can run migrations) - because you can have multiple containers in the task (nginx proxy), you should be able to specify the name of the container that you are intending to run the command in, defaulting to the name of the service (as we use it that way) - in case there is a different container name than service name (or you specify non-existent container name), this error is thrown for you to be able to set the correct container name parameter. 

I don't understand the confusion here.